Source/Core/VideoBackends/Vulkan/VulkanLoader.cpp

```diff
@@ -36,9 +43,9 @@ static void ResetVulkanLibraryFunctionPointers()
 
 static Common::DynamicLibrary s_vulkan_module;
 
-static bool OpenVulkanLibrary()
+static bool OpenVulkanLibrary(bool force_system_library)
```

Comment:
Can't we determine this if `customDriverLibraryName` is set?  Why do we need this variable too?

Reply:
`OpenVulkanLibrary` is also used to get information about the system graphics driver. In that case we want to use the system driver. `GpuDriverHelper::getSystemDriverInfo` does a JNI call which creates a Vulkan instance to retrieve some information about the driver.

Reply:
Yeah, I saw that.  What I was missing is that there were other calls as well.  There's a call to `LoadVulkanLibrary()` in `InitBackendInfo()` and `Initialize()` which is what calls this.

It's minor and I guess fine as it is.  But I feel like it might be a bit cleaner to have this take the library to load instead and if it is empty, it uses system.  Then instead of having two points of interaction (the bool + the library path) there is just one.

Reply:
For documentations sake:
I tried that version and we discussed it on Discord. I prefer the current approach and it's not a blocker.